app/views/guidances/admin_edit.html.erb

```diff
@@ -52,6 +52,6 @@
 <%= tinymce :content_css => asset_path('application.css') %>
 
 <!-- alert for the default template-->
-<div id="edit_guidance_alert_dialog" class="modal" style="display:none">
+<div id="edit_guidance_alert_dialog" class="modal" role="dialog" aria-label="<%=_("Missing Fields Alert")%>" style="display:none">
   <ul id="missing_fields_edit_guidance"></ul>

```

Comment:
I think this is ok for now. This ties into the larger issue of the site not having a consistent method for relaying form input errors.
Please make sure the focus gets set on the close button when the dialog opens.